src/pages/workspaces/workspace/analysis/Analyses.js

```diff
@@ -513,7 +514,7 @@ const Analyses = _.flow(
           refreshApps,
           uploadFiles,
           openUploader,
-          location,
+          googleBucketLocation,
```

Comment:
This code will need to pass either the `googleBucketLocation` or `azureContainerRegion`.